tests/MongoDB.Driver.Tests/ConnectionsSurvivePrimaryStepDownTests.cs

```diff
@@ -57,12 +59,19 @@ public class ConnectionsSurvivePrimaryStepDownTests
 
                 using (ConfigureFailPoint(client, errorCode))
                 {
-                    var exception = Record.Exception(() => { collection.InsertOne(new BsonDocument("test", 1)); });
+                    var document = new BsonDocument("test", 1);
```

Comment:
Don't cache the document to be inserted in a variable. See comment below.

Reply:
done